topics/jupyter-notebook/index.md

```diff
@@ -1,13 +1,15 @@
 ---
-created_by: Jupyter Developers
+aliases: notebook, jupyter, ipython-notebook, ipynb, ipynb-jupyter-notebook, ipynb-notebook
```

Comment:
I think `notebook` is too broad to include as an alias here. Looking through [the topic](https://github.com/topics/notebook), I see repositories for plenty of non-Jupyter projects, like [zadam/trilium](https://github.com/zadam/trilium).

The rest look good though 👍🏻 

```suggestion
aliases: jupyter, ipython-notebook, ipynb, ipynb-jupyter-notebook, ipynb-notebook
```

Reply:
Thanks, yes I agree